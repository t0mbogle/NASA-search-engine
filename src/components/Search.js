import React, { useState } from "react";
import "../styles/search.css";

const Search = () => {
    return(
        <>
          <input type="text" className="input" placeholder="search away :)" />
        </>
    );
};

export default Search;