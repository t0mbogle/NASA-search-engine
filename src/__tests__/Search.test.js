import React from "react";
import { render } from "@testing-library/react";
import Search from "../components/Search";

describe("Search", () => {
    const validProps = {
        setSearchResults: () => {},
    }

    it("renders correctly", () => {
        const { asFragment } = render(
          <Search 
            setSearchResults={validProps.setSearchResults}
          />
        );

        expect(asFragment()).toMatchSnapshot();
    });
});