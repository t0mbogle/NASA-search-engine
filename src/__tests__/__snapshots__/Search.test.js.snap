// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<DocumentFragment>
  <input
    class="input"
    placeholder="ello mate"
    type="text"
  />
</DocumentFragment>
`;
